packages/components/src/Tooltip/Tooltip.test.tsx

```diff
@@ -31,6 +34,59 @@ test("tooltip should show up on hover", () => {
   act(() => {
     getByTestId(contentID).dispatchEvent(new MouseEvent("mouseenter"));
   });
-  expect(getByText(message)).toBeTruthy();
-  expect(getByText(content)).toBeTruthy();
+  expect(getByText(message)).toBeInTheDocument();
+  expect(getByText(content)).toBeInTheDocument();
+});
+
+test("tooltip should show up on focus", () => {
```

Comment:
personal preference, should try to use `it` followed by a string that works like this:

```suggestion
it("should show up on focus", () => {
```

Reply:
this isn't a hard stop but might be useful to start a consistent approach

Reply:
ah same here. don't know why I missed that. haha. I'll update it